api/src/filters/validation-error.filter.ts

```diff
@@ -0,0 +1,34 @@
+import {
```

Comment:
Name is WTF. It's inside `filters` dir and you give `.filter` to file name. Otherwise the second file in this dir have `-filter` suffix. So nice combo.

Reply:
oo ye, i must fix it

Reply:
But on docs. they add .filter suffix, i create execeptions dir and inside put .filter files, i see this solution on 1 tutorial

Reply:
Okey, meybe You have right but it's still inconsistency one file is with `.filter` and in second file `-filter`.
Also link docs next time this will be good point to discussion.

Reply:
I fix it